classes/class_instance.lua

```diff
@@ -32,6 +32,8 @@ local modo_all = Details._detalhes_props["MODO_ALL"]
 local atributos = Details.atributos
 local sub_atributos = Details.sub_atributos
 
+local grouped_windows_horizontal_gap = 4
```

Comment:
This is bad. Local that can't be modified in any way and will severely disrupt user preferences

Reply:
I know, it's incomplete.